src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlAuthenticationParameters.cs

```diff
@@ -143,7 +143,7 @@ internal class Builder {
                 _authenticationMethod = authenticationMethod;
                 _serverName = serverName;
                 _databaseName = databaseName;
-                _scopes = new string[] { resource + "/.default" };
+                _resource = resource + "/.default";
```

Comment:
I think we should move this logic (appending "/.default") to ActiveDirectoryNativeAuthenticationProvider.cs and change it to only append if "/.default" is not already at the end of the string.